projects.json

```diff
@@ -1191,11 +1191,11 @@
     "compatibility": [
       {
         "version": "4.0",
```

Comment:
@msadoon
>I made the changes you suggested, but my precommit check for version 4.0 is failing locally with the same hash as for 5.0.
Do I need to rebuild the project in Xcode just for Swift 4.0 and make a commit? 

Looks like the new hash for your project is minimum Swift 4.2 compatible. Go ahead and bump the string here from `4.0` to `4.2` and I suspect your build will work

Reply:
Done! Works, thank you!